inyoka_theme_default/templates/planet/index.html

```diff
@@ -16,15 +16,20 @@
 ] %}
 
 {% block planet_content %}
+  {% set rendered_pagination = macros.render_pagination(pagination) %}
+
+  {{ rendered_pagination }}
   {% if days %}
     {% if page == 1 %}
       <div class="planet-newestarticles">
         <h2>{% trans %}Recent articles{% endtrans %}</h2>
-        <ul class="list-inline">
+        <ol class="planet-newestarticles-list">
```

Comment:
well, https://github.com/chris34/theme-default/commit/e655d01dbb3d4f8ed721908e9fa404cd406bab8f should cause this, so merge the autoprefixer-PR locally or checkout the commit before. And dont wonder if no numbers appear in Chrom{e,ium} neither after that, see the commit-mesage of https://github.com/chris34/theme-default/commit/a0e0cf6c22161e365a1b1e23ed18225cb3dcdae2 :D